Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -269,6 +283,32 @@ private extension HTTPClient {
         }
     }
 
+    struct ExponentialRetryOptions: CustomStringConvertible {
+
+        let retriableStatusCodes: Set<HTTPStatusCode>
+        let baseBackoff: TimeInterval
+        let maxBackoff: TimeInterval
+        let maxNumberOfRetries: UInt
+
+        var description: String {
+            """
+            <\(type(of: self)): retriableStatusCodes=\(retriableStatusCodes.debugDescription)
+            baseBackoff=\(baseBackoff.debugDescription)
+            maxBackoff=\(maxBackoff.debugDescription)
+            maxNumberOfRetries=\(maxNumberOfRetries)
+            """
+        }
+    }
+
+}
+
+internal extension HTTPClient.ExponentialRetryOptions {
+    static let `default` = HTTPClient.ExponentialRetryOptions(
```

Comment:
I'm totally open to changing these default values to something else if we want - this is just what came to the top of my head first!

Reply:
I wonder if it would make sense to have different defaults per endpoint type... As in, maybe we don't need to retry some non-critical requests like diagnostics, product entitlement mapping,...